README.md

```diff
@@ -1,36 +1,99 @@
+# xmtp-react-native
 
-Wrappers for native xmtp sdks for react native
+![Lint](https://github.com/xmtp/xmtp-android/actions/workflows/lint.yml/badge.svg) ![Status](https://img.shields.io/badge/Project_Status-Pre--Preview-red)
 
-## NOTES
+This repo provides a package you can use to build with XMTP in a React Native or Expo app.
 
-- To use the example app, you need to `npm install --force`
+> **Important:**  
+> This SDK is in **Pre-Preview** status and ready for you to start experimenting with.
+>
+> However, we do **not** recommend using Pre-Preview software in production apps. Software in this status will change as we add features and iterate based on feedback.
 
-# API documentation
+We are still working on adding several features to bring this SDK to parity with the [XMTP client SDK for JavaScript](https://github.com/xmtp/xmtp-js) (`xmtp-js`).
 
-- [Documentation for the main branch](https://github.com/expo/expo/blob/main/docs/pages/versions/unversioned/sdk/xmtp-react-native-sdk.md)
-- [Documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/xmtp-react-native-sdk/)
+Currently, these are the features supported by this SDK:
 
-# Installation in managed Expo projects
+- Create a `Client` from an ethers `Signer`
+- List conversations
+- List messages (unpaginated for now) in a conversation
+- Start a conversation
 
-For [managed](https://docs.expo.dev/versions/latest/introduction/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](#api-documentation). If you follow the link and there is no documentation available then this library is not yet usable within managed projects &mdash; it is likely to be included in an upcoming Expo SDK release.
+Here's a [list of features and issues](https://github.com/xmtp/xmtp-react-native/issues) we're still working on. 
 
-# Installation in bare React Native projects
+Is there a feature you need that's not on the list? Please [open an issue](https://github.com/xmtp/xmtp-react-native/issues).
 
-For bare React Native projects, you must ensure that you have [installed and configured the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/) before continuing.
+Or better yet, open a PR and we'll get it reviewed and merged as soon as possible. If you contribute a PR that gets merged into this repo, you'll be eligible to [claim this XMTP contributor POAP](https://www.gitpoap.io/gp/1042)!
 
-### Add the package to your npm dependencies
+To learn more about XMTP and get answers to frequently asked questions, see [FAQ about XMTP](https://xmtp.org/docs/dev-concepts/faq).
+
+![x-red-sm](https://user-images.githubusercontent.com/510695/163488403-1fb37e86-c673-4b48-954e-8460ae4d4b05.png)
+
+## Example apps
+
+This repo provides the following example apps that you can use to experiment and explore implementation details:
 
+- [Android example](example/android)
+
+- [iOS example](example/ios)
```

Comment:
I think the example app is just in example. There shouldn't be any need for them to go into the respective ios or android folders.